test/integration/utilsTest.ts

```diff
@@ -24,7 +26,7 @@ describe("Test: utils", function() {
         const ourHomeserver = new UserID(await this.mjolnir.client.getUserId()).domain;
         assert.equal(
             out.formatted_body,
-            `it's fun here in <a href="https://matrix.to/#/${config.managementRoom}?via=${ourHomeserver}">${config.managementRoom}</a>`
+            `it's fun here in <a href="https://matrix.to/#/${managementRoomAlias}?via=${ourHomeserver}">${managementRoomAlias}</a>`
```

Comment:
Hmm I don't think we're making these urls properly https://spec.matrix.org/v1.1/appendices/#matrixto-navigation I don't think it makes sense to inject `via` for a room with an alias. I also don't think we should depend on management rooms having an alias (at least outside of tests, this is a test).

Reply:
can this be addressed in a followup PR?